### PR TITLE
Milestone5

### PR DESCRIPTION
**Pull request by addod19**
## In this milestone, we performed the following actions:

- Review the design file from the given design and identify all the elements

- Implement the styling of the different components based on the design. Remember that each element needs to look exactly the same as in the design.

### To test our project, let us follow the instructions below:
- `git clone https://github.com/evansinho/React-Bookstore.git`
- `cd React-Bookstore`
- ` git checkout milestone5`
- `npm i`
- `yarn start`

A browser will open with our project as seen in the screenshot below

project

Kindly review our code :man_technologist:

### CODE REVIEW

**Comment by fatymahmed** [comment]
Friendly reminder: You should submit a working project, I followed the instructions in your readme file but it resulted to the error shown below.  
If you cannot fix it by yourself, reach out to a full-time TSE.   
[This article](https://microverse.zendesk.com/hc/en-us/articles/360026625973-How-can-I-get-help-from-TSEs-) describes how to set up a meeting with full-time TSEs.  
This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!  
> Uncaught TypeError: Cannot read property 'apply' of undefined

**Comment by addod19** [comment]
> Friendly reminder: You should submit a working project, I followed the instructions in your readme file but it resulted to the error shown below.
> If you cannot fix it by yourself, reach out to a full-time TSE.
> [This article](https://microverse.zendesk.com/hc/en-us/articles/360026625973-How-can-I-get-help-from-TSEs-) describes how to set up a meeting with full-time TSEs.
> This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!
> 
> > Uncaught TypeError: Cannot read property 'apply' of undefined

It is fixed!!! :+1: Thanks

File: README.md [hunk 1]
```diff
@@ -39,8 +39,8 @@ To get a local copy up and running follow these simple example steps.
 
 1. ``` git clone https://github.com/evansinho/React-Bookstore.git ```
 2. ``` cd React-Bookstore ```
-3. ``` git checkout  start```
-4. ``` yarn i ```
+3. ``` git checkout  milestone5```
+4. ``` npm i ```
```
Comment (bolah2009) [inline README.md]: Don't you think this instruction will be outdated after merging to your base (default) branch?

I would recommend removing the `git checkout  milestone5` option, this means when this branch is merged to your base branch, there won't be a need for you to checkout as the needed files would be available on the base branch.

File: src/index.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ import { Provider } from 'react-redux';
 import App from './components/App';
 import INITIAL_STATE from './constant/initialState';
 import rootReducer from './reducers';
+import './styles/index.scss';
 
 const STATE = { books: INITIAL_STATE };
```
Comment (bolah2009) [inline src/index.js:14]: Can you remove this code from your `createStore` parameters?
```js
    window.__REDUX_DEVTOOLS_EXTENSION__
                            && window.__REDUX_DEVTOOLS_EXTENSION__()));
```
This is resulting in an error shown below:
image


This is the same error reported by @fatymahmed earlier which means it's not from my side alone.


I can see it also affects [your live link ](https://shrouded-brushlands-72459.herokuapp.com/) in your Readme.

image

**Review by bolah2009 — APPROVED** [review]
# It's a 👍🏾 from me!

### The app looks like the given design. Check out the comments below before merging. :relaxed: 
image


### :heavy_check_mark: Approved!

Good luck with your next projects. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Pull request merged**